### ISSUE TITLE
Issues with older creation date not counted

### ISSUE THREAD

**Issue by JadenBalogh** [brandongk-ubco/gitstats#114]
GitStats only counts issues that were both created and closed within the given time interval. It should include all issues that were closed during the entered time frame, regardless of date created.

**Comment by brandongk-ubco** [comment]
Related to #100 - in short, the way @JadenBalogh expects it to work is the intended functionality.

@JadenBalogh does this happen all the time with all commits created before the range?  Can you confirm it is still happening with v1.0.10?

I am looking for steps to reproduce.

**Comment by JadenBalogh** [comment]
I attempted to put together a repro but hit the limit on API calls. At this moment, I can't confirm if this bug is still occurring, but I can test again once my API requests are off cooldown.

The simplest steps to reproduce _should_ be:
1. Create an issue and set it as a Task
2. Wait a few minutes, then close the issue
3. In GitStats, enter a date range starting between the creation and closing of the issue and ending after the closing

**Expected**: issue is included, as it was closed between the time interval
**Bug**: issue is not included, as it was *created* prior to the time range

So, if that repro is working as expected, either the issue is fixed, or the issue is more complex than initially guessed.
